enumerables.rb

```diff
@@ -0,0 +1,37 @@
+module MyEnumerable
+  def all?
+    @list.each { |num| return false unless yield(num) }
+    true
+  end
+
+  def any?
+    @list.each { |num| return true if yield(num) }
+    false
+  end
+
+  def filter
+    result = []
+    @list.each { |num| result << num if yield(num) }
+    result
+  end
+end
+
+class MyList
+  include MyEnumerable
+
+  def initialize(the_list)
+    @list = the_list
+  end
+
+  def each
+    @list.each
+  end
+end
+
+list = MyList.new([1, 2, 3, 4])
+
+puts(list.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+puts(list.filter(&:even?))
```

Comment:
Well done so far. However, kindly note that as part of the project requirements, each class and module should have a separate .rb file. Hence please consider separating them. :wink: 

![Screenshot from 2023-06-15 09-22-26](https://github.com/abeltsew/enumerables/assets/65233860/1ba0403e-6d4d-4a9f-853b-2de7611a1b56)